pubnub.gemspec

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Your rubocop was mad at me :-)

Reply:
@nevinera I can't find any warnings in Travis build logs. Is it possible to share what was wrong and why it has been required to add _freeze_?